<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridWindow">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWindow">
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabFile">
        <attribute name="title">
         <string>Files</string>
        </attribute>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Files</string>
         </property>
        </widget>
        <widget class="QTableView" name="tableView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>50</y>
           <width>321</width>
           <height>441</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabPaths">
        <attribute name="title">
         <string>Paths</string>
        </attribute>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Paths</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabConfig">
        <attribute name="title">
         <string>Config</string>
        </attribute>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Config</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabHelp">
        <attribute name="title">
         <string>Help</string>
        </attribute>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tabAbout">
        <attribute name="title">
         <string>About</string>
        </attribute>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>About</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_AddFiles"/>
    <addaction name="action_RemoveFiles"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Preferences">
    <property name="title">
     <string>&amp;Preferences</string>
    </property>
    <addaction name="action_Paths"/>
    <addaction name="action_Config"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Preferences"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_AddFiles">
   <property name="text">
    <string>&amp;Add Files</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_RemoveFiles">
   <property name="text">
    <string>Remove Files</string>
   </property>
  </action>
  <action name="action_Paths">
   <property name="text">
    <string>&amp;Paths</string>
   </property>
  </action>
  <action name="action_Config">
   <property name="text">
    <string>&amp;Config</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
